.github/workflows/build-test-package.yaml

```diff
@@ -33,7 +24,7 @@ jobs:
       - name: Assume AWS Validate role
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-region: ${{ env.REGION }}
+          aws-region: ${{ env.AWS_REGION }}
```

Comment:
Presume you've already checked this works, since you're changing it i.e. the AWS_REGION is definitely set?